pkg/kcl/api.go

```diff
@@ -408,7 +408,31 @@ func GetFullSchemaType(pathList []string, schemaName string, opts ...Option) ([]
 		return nil, err
 	}
 
-	return resp.SchemaTypeList, nil
+	result := []*gpyrpc.KclType{}
+	for _, kclType := range resp.SchemaTypeList {
+		if kclType.BaseSchema != nil {
+			baseSchema, err := getBaseSchemaRecursive(kclType.BaseSchema)
```

Comment:
I think it is not necessary to recursively obtain the base schema here because they are not directly dependent on each other.